pkgs/applications/editors/emacs/elisp-packages/manual-packages/jam-mode/default.nix

```diff
@@ -20,10 +21,19 @@ trivialBuild rec {
     runHook postUnpack
   '';
 
-  meta = with lib; {
+  patchPhase = ''
```

Comment:
Any reason to not use `postPatch`?

----------

I do not remember precisely the link, but the rationale for not customizing the whole `patchPhase` is because a custom patch phase does not account for the `patches = [ . . . ];` attribute (especially when it comes from a custom overlay).

Reply:
> Any reason to not use postPatch?

I guess `postPatch` is better to use in `overrideAttrs` and in the original nix expression it is better to use `patchPhase`.  Just a feeling.

> the rationale for not customizing the whole patchPhase is because a custom patch phase does not account for the patches = [ . . . ]; attribute

Oh, indeed.  My `patchPhase` does not take `patches` into account.